.github/workflows/update_version.yaml

```diff
@@ -0,0 +1,81 @@
+on:
+  schedule:
+    - cron: "0 9 */7 * *"
+jobs:
+  debug_info:
+    name: Debug info
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Print github context JSON
```

Comment:
Why do we need this?

Reply:
I was following from the ones that had been auto-gen. Happy to remove if you think it's not needed.

Reply:
if we don't use it I'd remove it, but your call 100%

Reply:
I'm gonna leave it for now while making sure the job actually works as expected. If all good then I'll remove :)